app/src/handlers/census/ContinentLockHandler.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ *  ### CENSUS RESPONSE ATTRIBUTES ####
+ "event_name":"",
+ "timestamp":"",
+ "world_id":"",
+ "zone_id":"",
+ "triggering_faction":"",
+ "previous_faction":"",
+ "vs_population":"",
+ "nc_population":"",
+ "tr_population":"",
+ "metagame_event_id":"",
+ "event_type":""
+ * ### END ###
+ **/
+
+import { injectable } from 'inversify';
+import EventHandlerInterface from '../../interfaces/EventHandlerInterface';
+import { GenericEvent } from 'ps2census/dist/client/utils/PS2Events';
+import { getLogger } from '../../logger';
+import config from '../../config';
+import { jsonLogOutput } from '../../utils/json';
+import ContinentLockEvent from './events/ContinentLockEvent';
+
+
+@injectable()
+export default class ContinentLockHandler implements EventHandlerInterface {
+    private static readonly logger = getLogger('ContinentLockHandler');
+
+    public handle(event: GenericEvent): boolean {
+        ContinentLockHandler.logger.debug('Parsing message...');
+        if (config.features.logging.censusEventContent) {
+            ContinentLockHandler.logger.debug(jsonLogOutput(event), { message: 'eventData'});
+        }
+        try {
+            const continentLockEvent = new ContinentLockEvent(event);
```

Comment:
Yeah, did not find anything I can do with it? You want to store it in the database?

Reply:
It would be reasonable to use this as an alert ender, it's baked into the game if the continent has locked and an alert is present the alert ends (or rather the reverse, the alert triggers the cont lock), so we'll probably need to add a check for any active alerts and force an alert end if this event triggers when the alert is still running.